### PR TITLE
Create pull_req_file.md

### PR DESCRIPTION
**Pull request by AdharaHand**
This ist ze pull request.

### CODE REVIEW

**Comment by AdharaOrigin** [comment]
Maaaaan. This sucks.

File: pull_req_file.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+## Hello, World!
+I am now your friend.
```
Comment (AdharaOrigin) [inline pull_req_file.md]: I'm not your friend!

File: pull_req_file.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
+## Hello, World!
```
Comment (AdharaOrigin) [inline pull_req_file.md]: I am not the World!

Comment (AdharaOrigin) [reply pull_req_file.md]: Hue hue hue...

File: pull_req_file.md (hunk 1 above)
Comment (AdharaHand) [reply pull_req_file.md]: Diz ok.

**Pull request merged**